src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -185,31 +185,34 @@ export default CrashHeader;
 const Wrapper = styled('div')`
   display: flex;
   justify-content: space-between;
-  flex-direction: column;
   flex-wrap: wrap;
   width: 100%;
-  @media (min-width: ${props => props.theme.breakpoints[2]}) {
-    align-items: center;
-    flex-direction: row;
-  }
 `;
 
-const TitleInfo = styled('div')`
+const TitleWrapper = styled('div')`
   display: flex;
-  flex: 1;
-  max-width: 100%;
-  flex-direction: column;
-  @media (min-width: ${props => props.theme.breakpoints[2]}) {
-    flex-direction: row;
-    flex-shrink: 2;
-    align-items: center;
+  flex-wrap: wrap;
+  align-items: center;
+  > * {
+    margin-bottom: ${space(0.5)};
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  @media (min-width: ${props => props.theme.breakpoints[0]}) {
+    > * {
+      margin-right: ${space(0.5)};
+      margin-bottom: 0;
+      :last-child {
+        margin-right: ${space(1)};
+      }
```

Comment:
Should this element use a grid layout to space out its children instead of margins?

Reply:
I have tried, but grid has some limitations (Please correct me if I am wrong)... I could use auto-fit  or auto-fill but I always have to inform a minimum width, something hard to predict as the content is dynamic. 

I also could have used media queries with grid-auto-rows: auto, but I didn't want to use media queries for something simple.